manager/manager.go

```diff
@@ -49,6 +49,16 @@ func (man *volumeManager) doCreate(volume *types.VolumeInfo) (*types.VolumeInfo,
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create volume '%s'", volume.Name)
 	}
+
+	if len(vol.Replicas) == 0 {
+		for i := 0; i < vol.NumberOfReplicas; i++ {
+			replica, err := man.orc.CreateReplica(vol.Name)
+			if err != nil {
+				return nil, errors.Wrapf(err, "error creating replica '%s', volume '%s'", replica.Name, vol.Name)
+			}
+			vol.Replicas[replica.Name] = replica
```

Comment:
Yeah, probably. I was assuming it was allocated during CreateVolume() call. We'd better check.